.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,14 +17,12 @@ What things should reviewers look out for?
 ### Formatting Checks
 * [ ] Removed any manual numbering of sections.
 * [ ] Removed any instances of chunk naming.
-* [ ] Spell checked any `.Rmd` file or md file.
 * [ ] Comments and documentation are up to date.
-* [ ] All links have been checked and are properly formatted. 
+* [ ] All links have been checked and are properly formatted.
 
 ### Add datasets to S3
 * [ ] Added [data and metadata files to S3](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#adding-datasets-to-the-S3-bucket).
 
 ### Docker/Snakemake rendering components
 * [ ] Added the `.html` [link to the navigation bar](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#add-new-analyses-to-the-navbar).
-* [ ] Any not yet added packages needed for this analysis have been added to the Dockerfile and it successfully builds.
-* [ ] In the Docker container, [snakemake was run for rendering](https://github.com/AlexsLemonade/refinebio-examples/blob/master/CONTRIBUTING.md#how-to-re-render-the-notebooks).
+* [ ] Any not yet added packages needed for this analysis have been added to the Dockerfile (this will be built and pushed by GitHub actions).
```

Comment:
I think I liked the previous wording better: People _should_ check that it successfully builds... It will be checked with the PR, but not pushed. (Push on merge to master)